src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -65,6 +76,9 @@ const StreamGroup = createReactClass({
     if (!valueIsEqual(this.state.data, nextState.data)) {
       return true;
     }
+    if (!valueIsEqual(this.state.showLifetimeStats, nextState.showLifetimeStats)) {
```

Comment:
Should we do just a normal boolean comparison this this value is a bool?

Reply:
Will dp